tmt/cli.py

```diff
@@ -174,9 +174,15 @@ def main(context, root, **kwargs):
     '-i', '--id', 'id_', help='Run id (name or directory path).', metavar="ID")
 @click.option(
     '-a', '--all', 'all_', help='Run all steps, customize some.', is_flag=True)
+@click.option(
+    '-e', '--environment', 'env_',
```

Comment:
The underscore in above options was used only because `all` and `id` are reserved in python. Let's use regular `environment` here.